std_msgs/Header header

string alg
bool start
float32[3] cur_position
float32[3] des_position
